.github/workflows/ci.yml

```diff
@@ -29,13 +47,31 @@ jobs:
           toolchain: stable
           components: clippy
 
-      - run: cargo clippy --workspace --all-features -- -D warnings
+      - run: cargo clippy -p ${{ matrix.crate }} --no-deps --all-features -- -D warnings
```

Comment:
This should improve check time and also not fail on dependency which has clippy issues.